api/src/main/resources/liquibase.xml

```diff
@@ -194,4 +194,26 @@
 			DELETE FROM person_attribute WHERE value IS NULL OR value = '';
 		</sql>
 	</changeSet>
+    <changeSet id="ugandaemr-20032018-1531" author="slubwama">
```

Comment:
@slubwama You do not need a scheduled task for this. An SQL Statement should do the trick as its only run once

Reply:
Sql statement was to complex and required multiple levels of looping or introducing new fuctions. The job disables it self after its done. So no worries that it will run again